test/unit/NestedFactory.unit.ts

```diff
@@ -1749,24 +1696,24 @@ describeWithoutFork("NestedFactory", () => {
             await expect(
                 context.nestedFactory
                     .connect(context.user1)
-                    .processOutputOrders(1, [
+                    .processOutputOrders(1, context.mockUSDC.address, expectedUsdcFees, [
                         { outputToken: context.mockUSDC.address, amounts: [uniSold, kncSold], orders, toReserve: true },
                     ]),
             )
                 .to.emit(context.nestedFactory, "NftUpdated")
                 .withArgs(1);
 
-            // 3 UNI must be in the reserve
+                // 3 UNI must be in the reserve
             expect(await context.mockUNI.balanceOf(context.nestedReserve.address)).to.be.equal(
                 baseUniBought.sub(uniSold),
             );
             // 0 KNC must be in the reserve
             expect(await context.mockKNC.balanceOf(context.nestedReserve.address)).to.be.equal(
                 baseKncBought.sub(kncSold),
             );
-            // 7 USDC - fees must be in the reserve
-            expect(await context.mockUSDC.balanceOf(context.nestedReserve.address)).to.be.equal(
-                usdcBought.sub(expectedUsdcFees),
+            // 7 USDC - fees must be payed by user
+            expect(await context.mockUSDC.balanceOf(context.user1.address)).to.be.equal(
```

Comment:
Is it no longer possible to check what the reserve has received ?

If it is still possible, it is good to test it.

Reply:
If I am not mistaken, it is checked in the next line
```solidity
// The FeeSplitter must receive the right fee amount (in USDC)
expect(await context.mockUSDC.balanceOf(context.feeSplitter.address)).to.be.equal(expectedUsdcFees);
```

Reply:
Yes, you are right, this is my mistake, I was confused by the `balanceOf(context.feeSplitter.address)`, I was expecting to have `balanceOf(context.nestedReserve.address)`.